crates/aptos-telemetry/src/build_information.rs

```diff
@@ -9,29 +9,31 @@ use std::collections::BTreeMap;
 const APTOS_NODE_BUILD_INFORMATION: &str = "APTOS_NODE_BUILD_INFORMATION";
 
 /// Build information keys
-const BUILD_BRANCH: &str = "build_branch";
-const BUILD_CARGO_VERSION: &str = "build_cargo_version";
-const BUILD_CHAIN_ID: &str = "build_chain_id";
-const BUILD_CLAP_VERSION: &str = "build_clap_version";
-const BUILD_COMMIT_HASH: &str = "build_commit_hash";
-const BUILD_OS: &str = "build_os";
-const BUILD_PKG_VERSION: &str = "build_pkg_version";
-const BUILD_PROJECT_NAME: &str = "build_project_name";
-const BUILD_RUST_CHANNEL: &str = "build_rust_channel";
-const BUILD_RUST_VERSION: &str = "build_rust_version";
-const BUILD_TAG: &str = "build_tag";
-const BUILD_TARGET: &str = "build_target";
-const BUILD_TARGET_ARCH: &str = "build_target_arch";
-const BUILD_TIME: &str = "build_time";
-const BUILD_VERSION: &str = "build_version";
+pub const BUILD_BRANCH: &str = "build_branch";
+pub const BUILD_CARGO_VERSION: &str = "build_cargo_version";
+pub const BUILD_CHAIN_ID: &str = "build_chain_id";
+pub const BUILD_CLAP_VERSION: &str = "build_clap_version";
+pub const BUILD_COMMIT_HASH: &str = "build_commit_hash";
+pub const BUILD_OS: &str = "build_os";
+pub const BUILD_PKG_VERSION: &str = "build_pkg_version";
+pub const BUILD_PROJECT_NAME: &str = "build_project_name";
+pub const BUILD_RUST_CHANNEL: &str = "build_rust_channel";
+pub const BUILD_RUST_VERSION: &str = "build_rust_version";
+pub const BUILD_TAG: &str = "build_tag";
+pub const BUILD_TARGET: &str = "build_target";
+pub const BUILD_TARGET_ARCH: &str = "build_target_arch";
+pub const BUILD_TIME: &str = "build_time";
+pub const BUILD_VERSION: &str = "build_version";
 
 // Get access to BUILD information
 shadow!(build);
 
 /// Collects and sends the build information via telemetry
 pub(crate) async fn create_build_info_telemetry_event(chain_id: String) -> TelemetryEvent {
     // Collect the build information
-    let build_information = get_build_information(Some(chain_id));
+    // Note: This is giving the build information of the telemetry crate and not the aptos node
```

Comment:
I think that was the original intent, but if clients want crate specific information, that makes sense too.